src/main/resources/db/migration/all/20240209155543__make_assessment_id_on_appeals_table_not_nullable.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE appeals
+SET assessment_id = assessments.id
+FROM applications
+JOIN assessments ON applications.id = assessments.application_id
+WHERE applications.id = appeals.application_id;
```

Comment:
is it possible for an application to have multiple assessments? If yes ... would this SQL fail in those cases (or just pick a random assessment)? Maybe we could make it more deterministic by choosing the newest assessment?